worldedit-core/src/main/java/com/boydti/fawe/config/Settings.java

```diff
@@ -487,8 +487,6 @@
         public int MODE = 1;
         @Comment({"If existing lighting should be removed before relighting"})
         public boolean REMOVE_FIRST = true;
-        @Comment({"Calculate and set heightmaps when relighting"})
-        public boolean DO_HEIGHTMAPS = true;
```

Comment:
It's probably worth keeping heightmaps as an optional thing rather than forcing onto each edit, similar to how lighting is configured.

Reply:
Heightmaps are essential for proper functionality, e.g. mob spawning, maps, block updates due to snow, ice forming etc. If we want to make it optional, we'd need a separat command to fix them. Otherwise, people end up with "broken" worlds.

Reply:
I always wanted to open an issue/feature request, but forgot - I think that heightmap updates, like light updates, should be enabled by default. However there _could_ be an option to turn it off (like with lighting) and also a seperate command, in addition to //fixlighting, "//fixheightmaps".

Reply:
As I've just said above, there are cases where heightmaps aren't necessarily needed, and thus the option to disable/have as a post-process might be ideal.